pkgs/rust-lang/rust-analyzer/registry.yaml

```diff
@@ -0,0 +1,21 @@
+packages:
+  - type: github_release
+    repo_owner: rust-lang
+    repo_name: rust-analyzer
+    asset: "rust-analyzer-{{.Arch}}-{{.OS}}.{{.Format}}"
```

Comment:
```suggestion
    asset: rust-analyzer-{{.Arch}}-{{.OS}}.{{.Format}}
```

https://aquaproj.github.io/docs/reference/registry-style-guide#remove-unneeded-quotes-of-strings